doc/website/blog/2023-05-08-atlas-v011-kubernetes-operator.md

```diff
@@ -107,7 +107,7 @@ operator, follow these steps to get started:
   to the database:
 
   ```bash
-  kubectl apply -f https://raw.githubusercontent.com/ariga/atlas-operator/master/config/integration/mysql.yaml
+  kubectl apply -f kubectl apply -f https://raw.githubusercontent.com/ariga/atlas-operator/65dce84761354d1766041c7f286b35cc24ffdddb/config/integration/databases/mysql.yaml
```

Comment:
```suggestion
  kubectl apply -f https://raw.githubusercontent.com/ariga/atlas-operator/65dce84761354d1766041c7f286b35cc24ffdddb/config/integration/databases/mysql.yaml
```

Reply:
Why not keep master? For testing or to tag this blog post to a specific version of the config?